scripts/README.md

```diff
@@ -62,6 +60,25 @@ The fleet size should be smaller than or equal to 20, in order to ensure that th
 Please document the displayed bracket-trader addresses. They are required for future withdrawals.
 They can also be retrieved from the created transactions. However, since this is a manual process, it is quite cumbersome to extract them right now.
 
+### Safe Airdrop
+
+Create your own transferFile (or use our sample `scripts/data/sampleTransferFile.json`)
+With a master safe that you own execute:
+
+```sh
+npx truffle exec scripts/airdrop.js --masterSafe 0xd9395aeE9141a3Efeb6d16057c8f67fBE296734c --transferFile scripts/data/sampleTransferFile.json --network rinkeby
```

Comment:
~~Master Safe is the universal term for this particular project (which was originally only for bracket strategies - aka CMM). When we migrate this "multisend" functionality into its own standalone repo, then I would change the name here.~~

Actually. I think this would be a good place to use the term `fundAccount`.